registry.md

```diff
@@ -7,32 +7,30 @@ This registry may be expanded with new RFCs.
+| Name     | Description            | Reference                            |  `value`   | `parameters` key | `parameters` value |`parameters` description         |
+|:---      |:---                    |:---                                  |:---        |:---              |:---                |:---                             |
+| Bitcoin  | Bitcoin-core network   | [RFC-004](./RFC-004-SWAP-Bitcoin.md) | `bitcoin`  | `network`        |                    | The network on which to operate |
+|          |                        |                                      |            |                  | `regtest`          | Bitcoin-core regtest            |
+|          |                        |                                      |            |                  | `testnet`          | Bitcoin testnet                 |
+|          |                        |                                      |            |                  | `mainnet`          | Bitcoin mainnet                 |
+| Ethereum | Ethereum network       | TBD                                  | `ethereum` | `network`        |                    | The network on which to operate |
+|          |                        |                                      |            |                  | `regtest`          | Local dev network               |
+|          |                        |                                      |            |                  | `ropsten`          | Ropsten testnet (network id 3)  |
+|          |                        |                                      |            |                  | `mainnet`          | Ethereum mainnet (network id 1) |
 
 
 [RFC003](./RFC-003-SWAP-basic.md#identity) requires that each ledger has an associated identity:
 
-| Ledger   | Identity Name | JSON Encoding            | Reference | Description                                                                       |
-|:----     |:-------      |:-------------            |:--------- | --------------------------------------------------------------------------------- |
-| Bitcoin  | `pubkeyhash`  | `hex-encoded-bytes (20)` | TBD       | The result of applying SHA256 and then RIPEMD160 to a user's SECP256k1 public key |
-| Ethereum | `address`     | Ethereum address        | TBD       | An Ethereum address                                        |
+| Ledger   | Identity Name | JSON Encoding            | Reference                            | Description                                                                                  |
+|:----     |:-------      |:-------------             |:---------                            | ---------------------------------------------------------------------------------            |
+| Bitcoin  | `pubkeyhash`  | `hex-encoded-bytes (20)` | [RFC-004](./RFC-004-SWAP-Bitcoin.md) | The result of applying SHA256 and then RIPEMD160 to a user's SECP256k1 compressed public key |
```

Comment:
:wrench: The sentence is already quite long, maybe this can be omitted?

```suggestion
| Bitcoin  | `pubkeyhash`  | `hex-encoded-bytes (20)` | [RFC-004](./RFC-004-SWAP-Bitcoin.md) | The result of applying SHA256 and then RIPEMD160 to a SECP256k1 compressed public key |
```